src/main/java/frc/robot/commands/MoveElevatorRecordMaxHeight.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.commands;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.constants.ElevatorConstants;
+import frc.robot.subsystems.Elevator;
+
+public class MoveElevatorRecordMaxHeight extends CommandBase {
+  Elevator m_elevator; 
+
+  public MoveElevatorRecordMaxHeight(Elevator elevator) {
+    m_elevator = elevator; 
+    addRequirements(m_elevator);
+  }
+
+  @Override
+  public void initialize() {
+    m_elevator.set(ElevatorConstants.kElevatorMotorEncoderZeroingPower);
+  }
+
+  @Override
+  public void end(boolean interrupted) {
+    m_elevator.stopMotor();
+    ElevatorConstants.kElevatorTopHeightMeters = m_elevator.getElevatorHeightMeters();
```

Comment:
If you are changing the value, it is probably not a constant, maybe it should just be a field in Elevator.java

Reply:
Change has been incorporated.